cfgov/jinja2/v1/youth_employment_success/index.html

```diff
@@ -15,14 +15,15 @@
     <section class="content_main">
       <h1>Plan options for getting to work</h1>
       {% include "budget-form.html" %}
-      <div class="yes-routes o-expandable-group o-expandable-group__accordion">
+      <div class="yes-routes">
         <div class="content-l">
-          {% for i in range(3) %}
+          {% for i in range(2) %}
             {% with id=i+1 %}
               {% include "route-option.html" %}
             {% endwith %}
           {% endfor %}
         </div>
+        <button class="m-yes-route-option">Show me another option</button>
```

Comment:
Actually no, in the mockups it looks like a link with an icon. I'm basically not worrying about styling at this point though, as there is a lot of JS functionality to add

Reply:
Here is what the button should look like:

![Screen Shot 2019-08-30 at 11 19 47 AM](https://user-images.githubusercontent.com/1421848/64043033-26ac7200-cb18-11e9-9a81-6faaa3f580b4.png)

It will obviously still have the `role=button`